go/libkb/chain_link_v2.go

```diff
@@ -57,6 +57,7 @@ const (
 	// Note that 45 is skipped, since it's retired; used to be LegacyTLFUpgrade
 	SigchainV2TypeTeamSettings     SigchainV2Type = 46
 	SigchainV2TypeTeamKBFSSettings SigchainV2Type = 47
+	SigchainV2TypeTeamBotSettings  SigchainV2Type = 48
```

Comment:
`IsSupportedTeamType`

maybe `TeamAllowStub`. I'm surprised that other settingsish types aren't in there.